framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1902,8 +1902,8 @@ private AnnotatedTypeMirror fromExpression(ExpressionTree tree) {
      * #addComputedTypeAnnotations(Element, AnnotatedTypeMirror)}.
      *
      * <p>In classes that extend {@link GenericAnnotatedTypeFactory}, override {@link
-     * GenericAnnotatedTypeFactory#addComputedTypeAnnotations(Tree, AnnotatedTypeMirror, boolean)}
-     * instead of this method.
+     * GenericAnnotatedTypeFactory#addComputedTypeAnnotations(Tree, AnnotatedTypeMirror)} instead of
```

Comment:
This comment no longer makes sense, right? That is the same method now, as the extra parameter was removed. Should something in this class be done differently now?

Reply:
Yes, I think so. I will remove this comment.